packages/relay/package.json

```diff
@@ -5,8 +5,11 @@
   "main": "index.js",
   "dependencies": {
     "@kiwicom/margarita-graphql": "^0",
-    "@kiwicom/relay": "^0.1.0",
-    "react-relay": "^1.7.0"
+    "@kiwicom/relay": "^0.6.0",
+    "relay-runtime": "^2.0.0",
+    "react-relay": "^2.0.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "babel-plugin-relay": "^2.0.0"
+  }
```

Comment:
This should all be removed and you should keep only `"@kiwicom/relay": "^0.8.0",`. The babel plugin may be necessary if you do not use `@kiwicom/babel-preset-kiwicom` (which I highly recommend).

Reply:
I will look into this in a separate PR, since it seems to crash all our tests.

Reply:
Ok, looks like I solved it, so I will add it to this PR.